### PR TITLE
optionally override cms hostname in nginx

### PR DESCRIPTION
**Pull request by fredsmith**

### CODE REVIEW

File: playbooks/roles/nginx/templates/edx/app/nginx/sites-available/cms.j2 [hunk 1]
```diff
@@ -27,8 +27,11 @@ server {
   listen {{EDXAPP_CMS_NGINX_PORT}} {{default_site}};
   {% endif %}
 
-
+  {% if cms_hostname is defined %}
```
Comment (feanil) [inline playbooks/roles/nginx/templates/edx/app/nginx/sites-available/cms.j2]: If this is an override, we should make it an all caps variable.

**Comment by e0d** [comment]
why not just override the var rather than have conditional logic in the template.

**Comment by jarv** [comment]
:+1:

**Comment by jarv** [comment]
@fredsmith Yeah I think I agree with @e0d and would put `~^((stage|prod)-)?studio\..*` in a var

**Comment by fredsmith** [comment]
Ah, that's a good idea.  thanks!

**Comment by fredsmith** [comment]
can I get another :+1: on this?

**Comment by feanil** [comment]
:+1:

**Pull request merged**